tools/condor-spectate-flarm.py

```
@@ -0,0 +1,605 @@
+             remote_name: str) -> bytes:
+    """Read a share file read-only via a short-lived SMB session.
+
+    Uses ``get remote -`` (stdout only, no local copy). Each poll opens,
+    reads, and closes immediately so Condor can keep writing Spectate.json.
+    """
+    url = f"//{host}/{share}"
+    cmd = [
+        "smbclient", url, "-U", f"{user}%{password}",
+        "-c", f"get {remote_name} -",
+    ]
+    result = subprocess.run(cmd, check=True, stdout=subprocess.PIPE,
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)